packages/styles/index.html

```diff
@@ -31,6 +31,65 @@ <h1>Styles package playground</h1>
     <hr />
 
     <!-- Place your component markup here -->
-    <button class="btn btn-primary">Primary button</button>
+
+    <h3>.Validated</h3>
+    <form
+      class="form-control container was-validated"
+      style="margin: 24px; max-width: calc(100vw - 48px)"
+      data-toggle="validator"
+    >
+      <div class="form-text">
+        <label class="form-label" for="111">Label</label>
+        <input
+          id="111"
+          class="form-control form-control-lg"
+          type="text"
+          placeholder="Placeholder"
+          required
+        />
+        <div class="form-text">
+          Hintus textus elare volare cantare hendrerit in vulputate velit esse molestie consequat,
+          vel illum dolore eu feugiat nulla facilisis.
+        </div>
+        <div class="valid-feedback">Looks good!</div>
+        <div class="invalid-feedback">Please provide a valid input.</div>
+      </div>
+      <div class="mb-3">
+        <select class="form-select" aria-label="select example" required>
+          <option value="">Open this select menu</option>
+          <option value="1">One</option>
+          <option value="2">Two</option>
+          <option value="3">Three</option>
+        </select>
+        <p class="valid-feedback">You’ve selected an option.</p>
+        <p class="invalid-feedback">Please select an option.</p>
+      </div>
+      <fieldset class="form-check" required>
```

Comment:
Did you find this fieldset structure in the docs? If so, what is the link?

Reply:
i think i took it from the documentation from the inline example :https://design-system.post.ch/?path=/docs/151242aa-a074-4a55-a81c-db597c83cdad--docs